apps-rendering/src/components/Layout/GalleryLayout.tsx

```diff
@@ -0,0 +1,54 @@
+// ----- Imports ----- //
+import { css } from '@emotion/react';
+import { neutral } from '@guardian/source-foundations';
+import Headline from 'components/Headline';
+import MainMedia, { GalleryCaption } from 'components/MainMedia';
+import Metadata from 'components/Metadata';
+import Series from 'components/Series';
+import Standfirst from 'components/Standfirst';
+import { grid } from 'grid/grid';
+import type { Item } from 'item';
+import { getFormat } from 'item';
+import type { FC } from 'react';
+
+// ----- Component ----- //
+
+const headerStyles = css`
+	${grid.container}
+	background-color: ${neutral[7]};
+	border-bottom: 1px solid ${neutral[20]};
+`;
+
+const wrapperStyles = css`
+	background-color: ${neutral[7]};
+`;
+
+type Props = {
+	item: Item;
+};
+
+const GalleryLayout: FC<Props> = ({ item, children }) => (
+	<main>
+		<article css={wrapperStyles}>
+			<header css={headerStyles}>
+				<MainMedia
+					mainMedia={item.mainMedia}
+					format={getFormat(item)}
+				/>
+				<Series item={item} />
+				<Headline item={item} />
+				<Standfirst item={item} />
+				<Metadata item={item} />
```

Comment:
I may have missed it (if so let me know!), but I can't see anywhere that we're rendering a byline? Should that be included in the metadata component?

Reply:
No you're absolutely right and now you've pointed that out it does seem like a glaring omission. However, I can't see a byline in the Dotcom galleries either so that's how that happened. Who do I talk to about this?

Reply:
I suspect not all galleries have one, but many do. Here's an example: https://www.theguardian.com/artanddesign/gallery/2022/jul/22/the-week-around-the-world-in-20-pictures

![gallery-byline](https://user-images.githubusercontent.com/53781962/180756930-8ddedb1f-fc2c-44ad-8f86-a8d6586d4bf5.jpg)

Reply:
Brilliant, thanks for digging that out, I'll make sure that gets added in